logging_loki/handlers.py

```diff
@@ -64,12 +64,14 @@ def emit(self, record: logging.LogRecord):
         # noinspection PyBroadException
         try:
             labels = self.build_labels(record)
-            ts = rfc3339.format(record.created)
+            ts = rfc3339.format_microsecond(record.created)
             line = self.format(record)
             payload = {"streams": [{"labels": labels, "entries": [{"ts": ts, "line": line}]}]}
             resp = self.session.post(self.url, json=payload)
             if resp.status_code != 204:
-                raise ValueError("Unexpected Loki API response status code: %s" % resp.status_code)
+                raise ValueError(
+                    f"Unexpected Loki API response status code: {resp.status_code} : {resp.text}"
```

Comment:
In setup.py requirement for python is `>=3.5`, although this is not specified in classifiers. f-string appeared in python 3.6 and I'm don't want to drop support for 3.5. Please, format string with modulo formatter or with `.format(...)` method.

Reply:
Also, the response text can be very large. 
I think it would be better to display `text[:self.response_text_log_size]`, where `response_text_log_size` is a class variable with the default value of 1024.